src/satellitePropagator/physics/YoshidaIntegrator.h

```diff
@@ -0,0 +1,297 @@
+//
+// Created by FG-TUM on 17.02.22.
+//
+
+#pragma once
+#include <cmath>
+
+#include "AccelerationAccumulator.h"
+#include "satellitePropagator/debris/AccelerationUpdate.h"
+/**
+ * @class Integrator
```

Comment:
Docstring not up-to-date , we might wanna add a reference to the paper here? And, possibly, that it is the 4th order version from wiki?

Reply:
Then please add a suggestion here ;)